dockers/docker-platform-monitor/Dockerfile.j2

```diff
@@ -27,17 +26,18 @@ RUN apt-get update &&   \
         libpci3         \
         iputils-ping    \
         pciutils        \
+        ethtool &&	\
```

Comment:
@ArunSaravananBalachandran why is the ethtool needed?

Reply:
ethtool is already being installed in pmon and has not been added as part of this PR.